demo/src/Components/Cards/StatusCards/index.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react'
+import classNames from 'classnames'
+
+// import Icon from '../Icon'
+import Text from '../Text'
+import Title from '../Title'
+
+class StatusCards extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      status: {
+        success: true,
```

Comment:
I know I commented this before but it seems to have been buried.  We don't need state here as we are dealing with static data (classnames) 
```
const StatusCards = () => (
  <section className="o-section-md">
     <div className="o-site-wrap o-site-wrap--padding">
       <h4>Cards: Status Cards</h4>
       <div className="o-grid o-grid--gutter-md o-grid--matrix">
         <div className="o-grid__item u-1/2@sm">
            <div className="c-card c-card--success">
              <h4>Success</h4>
              <p>Lorem ipsum...</p>
            </div>
         </div>
         <div className="o-grid__item u-1/2@sm">
           <div className="c-card c-card--success">
             <h4>Success</h4>
             <p>Lorem ipsum...</p>
           </div>
         </div>
         <div className="o-grid__item u-1/2@sm">
           <div className="c-card c-card--success">
             <h4>Success</h4>
             <p>Lorem ipsum...</p>
           </div>
         </div>
         <div className="o-grid__item u-1/2@sm">
           <div className="c-card c-card--success">
             <h4>Success</h4>
             <p>Lorem ipsum...</p>
           </div>
         </div>
       </div>
     </div>
  </section>
)
```